source/command_bus.coffee

```diff
@@ -7,19 +7,22 @@ class Space.messaging.CommandBus extends Space.Object
   }
 
   _handlers: null
+  _onSendHooks: null
 
   constructor: ->
     super
     @_handlers = {}
+    @_onSendHooks = []
 
-  send: (command, callback) ->
+  send: (command, callback, options={}) ->
     if @meteor.isServer
       # ON THE SERVER
       handler = @_handlers[command.typeName()]
-      if not handler?
+      hook(command) for hook in @_onSendHooks when !options.ignoreHooks
+      if !handler? and !options.silent
```

Comment:
Yeah this is related – if `options.silent` is true, there is no exception thrown even when there is no handler for a command.